### ISSUE TITLE
getNegotiatedLanguageString() on LanguageMap

### ISSUE THREAD

**Issue by jbajou** [RusticiSoftware/TinCanPHP#76]
Hi guys,

just a quick question for you. Calling `getNegotiatedLanguageString()` on a LanguageMap throws an Exception (when throwing another exception):

```
Class 'TinCan\BadMethodCallException' not found
```
I'm trying to do the following:
```php
$statement->getTarget()->getDefinition()->getName()->getNegotiatedLanguageString()
```
Am I missing something ?
Thanks a lot

**Comment by brianjmiller** [comment]
I believe this should be fixed in https://github.com/RusticiSoftware/TinCanPHP/commit/c845d8fc5d569ccbcab25c3a2550344dc3db005d#diff-b5558596d61c6c61a22b0c8329b597bf which landed in release 0.50.1. What version are you on?

**Comment by jbajou** [comment]
Oh god.. Sorry I was using a version older than my grand mother (0.12 so you can laugh).
I just updated to 1.0.* and it works like a charm.
Thanks for your help, and sorry again
